includes/class-gs-faq-shortcode.php

```diff
@@ -38,6 +38,7 @@ function shortcode( $atts ) {
 			'id'    => '',
 			'cat'   => '',
 			'limit' => -1,
+			'order' => 'DESC'
```

Comment:
```suggestion
			'order' => 'DESC',
```

According to WPCS, the last item of an array should have a [comma](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#indentation)
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#indentation

Reply:
I'll commit it anyway and fix the issue in the `develop` branch.